README.md

```diff
@@ -1,2 +1,242 @@
-# eeajs
-EEA JavaScript libraries.
+# EEA JavaScript libraries - EEAJS
+EEA JavaScript libraries to interact with Pantheon-Quickstart exploring the privacy feature nd deploy contracts.
```

Comment:
nd -> and

Reply:
Would be good to add a section mentioning you can use the scripts in the multinode example with your own Pantheon nodes by modifying the `keys.js` file.  Could also include a link back to the Pantheon docs for information on how to configure your own nodes with Privacy (https://docs.pantheon.pegasys.tech/en/latest/Privacy/Configuring-Privacy/)

Reply:
Something like: https://hackmd.io/oWNddP-CQXq0QB1YDBTzPA